sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -642,13 +642,13 @@ export type RegistrationType = "Adm" | "AdmTemplate" | "Apple" | "AppleTemplate"
 
 // @public
 export interface ScheduleNotificationOptions extends OperationOptions {
-    tags?: string | string[];
+    tagExpression?: string;
```

Comment:
`createTagExpression` is not exported? I wonder whether we should have a special type `TagExpression` to prevent users from passing in arbitrary strings.